src/jesse_state.erl

```diff
@@ -192,7 +192,7 @@ resolve_ref(State, Reference) ->
       Path = jesse_json_path:parse(Pointer),
       case load_local_schema(State#state.root_schema, Path) of
         ?not_found ->
-          jesse_error:handle_schema_invalid(?schema_invalid, State);
+          jesse_error:handle_schema_invalid(Reference, State);
```

Comment:
isn't the `$ref` already part of the error?
I've cherry-picked your change, and instead output the canonical reference (the output from `combine_id`). That is slightly more valuable i.e. see how the `$ref` was expanding, and see what jesse tried to find and load.